frontend/src/components/pages/search/Search.tsx

```diff
@@ -71,22 +81,23 @@ const Search: React.FC = () => {
     apiService(dataType.filter, requestBody).then(users => setUsers(users));
   };
 
-  const years = [...Array(hightLimit - lowLimit)].map((_, index) => index + lowLimit);
+  const lowYears = [...Array(hightLimit - lowLimit)].map((_, index) => index + lowLimit);
+  const hightYears = [...Array(hightLimit - lowLimit)].map((_, index) => hightLimit - index);
```

Comment:
no need to do the same, maybe we can const hightYears = [... lowYears].reverse()?